### DISCUSSION TITLE
Github auto reject me form the student program for no known reason

### DISCUSSION THREAD

**Discussion by shacharrr** [community/community#27657] [GitHub Education] [answered]
I learn computer science at my high school in Israel, and I have tried applying to GitHub student plan for 5 time already, I used my school email, and even created new account to see if the problem was in the account, but every time the GitHub Bot auto reject me in the first 5 seconds, and it repeats twice that I need to complete my GitHub account, which the link send me to GitHub Docs and just explain me about how to customize my account. The weird problem is that my friend from school sent exact same type of verification and was verified, yet I got rejected in the first 5 seconds for 5 times already, Do you know what can be the problem?

**Comment by FahdSeddik** [comment, marked as answer]
Hey!
You have to make sure that your school's email is applicable for getting the benefits. Ask someone from administration. Hope it helps!

**Reply by shacharrr** [reply]
Ye, the email is fine, my friend used same type of school email and it worked for him perfectly

**Reply by FahdSeddik** [reply]
Then check out this [link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/why-wasnt-my-application-for-a-student-developer-pack-approved). If that does not help either then contact support [here](https://support.github.com/contact/account).

**Reply by FahdSeddik** [reply]
Mark it as an answer so people would see it easier.

**Reply by shacharrr** [reply]
Ok, after some time, now I got university id and now github accepted me, for some reason github didnt accept my school id.
